src/Language/PureScript/Make/Monad.hs

```diff
@@ -64,34 +64,34 @@ runMake opts = runLogger' . runExceptT . flip runReaderT opts . unMake
 -- | Run an 'IO' action in the 'Make' monad. The 'String' argument should
 -- describe what we were trying to do; it is used for rendering errors in the
 -- case that an IOException is thrown.
-makeIO :: Text -> IO a -> Make a
+makeIO :: (MonadIO m, MonadError MultipleErrors m) => Text -> IO a -> m a
```

Comment:
Are we using these at types other than `Make`? I'm not necessarily opposed to generalising them, but I think the doc-comments could probably do with updating if we are going to do it.

Reply:
Yes the IDE is using the functions to hash files and write/read the cachedb from here.